src/components/Nav/Nav.tsx

```diff
@@ -44,13 +44,15 @@ export class Nav extends React.Component<INavProps, INavState> {
   }
 
   private _renderLink(link: INavLink, linkIndex: number): React.ReactElement<{}> {
+    let { onLinkClick, shouldAssignTitle = true } = this.props;
     return (
       <li key={ linkIndex }>
         <a
           className={'ms-Nav-link' + (_isLinkSelected(link) ? ' is-selected' : '')}
           href={ link.url || 'javascript:' }
-          onClick={ this.props.onLinkClick }
-          title={ link.name }
+          onClick={ onLinkClick }
+          title={ shouldAssignTitle ? link.name : '' }
```

Comment:
Did you confirm this skips adding title altogether or makes for an empty tooltip? If the latter, consider using undefined in the else.